test/sqllogictest/outer_join_simplification.slt

```diff
@@ -676,46 +572,29 @@ select a, -2, 'keys', a from keys;
 ----
 Explained Query:
```

Comment:
I think this query is missing something, because more of it should have been removed. All of `c0` looks empty, at which point `keys` is empty also, and `bar2` can be seen to be empty. I'll investigate.

Reply:
Ah, found it. The analysis is correct, but the transform bails out on `LetRec` other than for the body.